modules/caddyhttp/server.go

```diff
@@ -1056,9 +1081,13 @@ const (
 )
 
 var networkTypesHTTP3 = map[string]string{
-	"unix": "unixgram",
-	"tcp4": "udp4",
-	"tcp6": "udp6",
+	"unixgram": "unixgram",
+	"udp":      "udp",
+	"udp4":     "udp4",
+	"udp6":     "udp6",
+	"tcp":      "udp",
+	"tcp4":     "udp4",
+	"tcp6":     "udp6",
```

Comment:
Do we still want "unix" to map to "unixgram"?

Reply:
in my opinion, no. a unix socket can only serve either h1/h2 xor h3, so unlike an tcp address implicitly serving h3 on the udp address with the same host:port, I think of a unix listener serving h3 as an incorrect configuration. just like it would be incorrect to serve h1/h2 from a unixgram address.

Reply:
Hm, okay that makes sense. Thanks!